apps/trading-e2e/src/integration/accounts.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Accounts table 
+  Scenario: Accounts displayed when connected to wallet
+    Given I am on the trading page for an active market
+    And I connect to Vega Wallet
+    When I click on accounts tab
+    Then accounts are displayed
+    And I can see account for tEURO
```

Comment:
Question (non-blocking): What do you think about moving the tests for the trading tabs (accounts/orders/positions etc.) into a single trading page feature file? 
It's quite expensive time wise to have lots of feature files. Cypress tears down and reloads an instance of Chrome for each feature file in a test run so I'd prefer to have as less as possible - as long as it makes sense.

Reply:
This will become doubly true when capsule is a thing and we have to remove parallelisation from the tests (hopefully we can have it back at a later date)